mandatory/1-fix-functions.js

```diff
@@ -10,31 +10,28 @@
 
 */
 
-function mood() {
+function mood(happy) {
   let isHappy = true;
 
-  if (isHappy) {
+  if (happy === isHappy) {
```

Comment:
Hi Gulnihaln, your overall code is good. However I think you can write only " if (isHappy)"  if you have (isHappy) as the parameter of the mood function instead of having a separated variable. Ex:
function mood(isHappy) {
  if (isHappy) {
    return "I am happy";
  } else {
 "I am not happy"
}
}

Reply:
Thank you Konika, I don't know the problem but this is what I tried and failed. I've tried it again now, it failed again.

Reply:
Thanks Konika, I tried it before but I've found my mistake now, when I wrote it in this way.